sdk/cosmos/azure-cosmos/README.md

```diff
@@ -428,6 +432,83 @@ print(json.dumps(container_props['defaultTtl']))
 
 For more information on TTL, see [Time to Live for Azure Cosmos DB data][cosmos_ttl].
 
+### Using the asynchronous client
+
+The asynchronous cosmos client looks and works in a very similar fashion to the already existing client, with the exception of its package within the sdk and the need of using async/await keywords in order to interact with it.
+
+```Python
+from azure.cosmos.aio import CosmosClient
+import os
+
+url = os.environ['ACCOUNT_URI']
+key = os.environ['ACCOUNT_KEY']
+client = CosmosClient(url, credential=key)
+database_name = 'testDatabase'
+database = client.get_database_client(database_name)
+container_name = 'products'
+container = database.get_container_client(container_name)
+
+async def create_items():
+    for i in range(1, 10):
+        await container.upsert_item({
+                'id': 'item{0}'.format(i),
+                'productName': 'Widget',
+                'productModel': 'Model {0}'.format(i)
+            }
+        )
+    await client.close()
+```
+
+It is also worth pointing out that the asynchronous client has to be closed manually after its use, either by initializing it using async with or calling the close() method directly like shown above.
```

Comment:
@simorenoh 
> closed manually after its use
Does it need to be closed after every function it's used in (like above), or can the client be passed around safely when doing things async?

Reply:
It can be passed around safely, but needs to be manually disposed of once you're done using it - the alternative to this being to use the client using a `with` statement:
`async with CosmosClient(url, key) as client:` and then starting your cosmos db logic within that context

Reply:
@simorenoh  What happens if these async clients are not closed properly? Memory leak or something else?

Reply:
I would have to test, but I assume that is the case - you also get an error in your code stating the client was not properly disposed of